packages/drivers/odsp-driver/src/prefetchLatestSnapshot.ts

```diff
@@ -111,7 +126,15 @@ export async function prefetchLatestSnapshot(
 				putInCache,
 				removeEntries,
 				enableRedeemFallback,
-			);
+			)
+				.then((value) => {
+					assert(!!snapshotEpoch, "prefetched snapshot should have a valid epoch");
+					snapshotContentsWithEpochP.resolve({ ...value, fluidEpoch: snapshotEpoch });
+				})
+				.catch((err) => {
+					snapshotContentsWithEpochP.reject(err);
+					throw err;
+				});
 			assert(cacheP !== undefined, 0x1e7 /* "caching was not performed!" */);
```

Comment:
I believe you need to remove cache line from cache after request is done.
If you do not do so, then all future loads will keep using that cache line.
Instead, we should rely on persistent cache (after load finished), and overall load flow of continuously (on each load) refreshing permanent cache.

Reply:
@vladsud What do you mean by removing cache line?

Reply:
You mean once the response is stored in permanent cache, then we should remove it from the non persistent cache?

Reply:
Removed the promise from the non-persistent cache, once it is stored in the persistent cache.